### PR TITLE
Add tables aws_inspector2_coverage, aws_inspector2_coverage_statistics, aws_inspector2_finding, and aws_inspector2_member (#1657)

### PR DESCRIPTION
**Pull request by misraved**
# Integration test logs
<details>
  <summary>Logs</summary>

```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>

```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

**Review by misraved — COMMENTED** [review]
@ParthaI please take a look at the review comments. Thanks!!

File: aws/table_aws_inspector2_coverage.go [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+		GetMatrixItemFunc: SupportedRegionMatrix(inspector2v1.EndpointsID),
+
+		// We *do not* use the common columns, because the account_id/region of
+		// the default columns come from the call, *not* from the retutned data.
+		// For inspector2, the account_id or region can vary within a single
+		// call.
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				// The account id from the data, rather than from the call (getCommonColumns).
+				Name:        "resource_account_id",
```
[the remark this answers is not in the document]
Comment (ParthaI) [reply aws/table_aws_inspector2_coverage.go:50]: Updated `resource_account_id` to `source_account_id`.

File: aws/table_aws_inspector2_coverage.go [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+				Name:        "ecr_image_tags_src",
+				Description: "Tags associated with the Amazon ECR image metadata.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ResourceMetadata.EcrImage.Tags"),
+			},
+			{
+				Name:        "ecr_image_tags",
+				Description: "Tags associated with the Amazon ECR image metadata.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ResourceMetadata.EcrImage.Tags"),
```
[the remark this answers is not in the document]
Comment (ParthaI) [reply aws/table_aws_inspector2_coverage.go:101]: Updated the ECR image tags columns as per the optional key qual and the API response.

File: aws/table_aws_inspector2_coverage_statistics.go [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+// value.
+//
+// Note that *if* it is decided to add dummy columns so as to enable filtering
+// (and thus counts_by_group), the filter columns are exactly the same ones as
+// in inspector2_coverage, and could be refactored/shared.
+func tableAwsInspector2CoverageStatistics(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_inspector2_coverage_statistics",
+		Description: "AWS Inspector2 Coverage Statistics",
+		List: &plugin.ListConfig{
```
[the remark this answers is not in the document]
Comment (ParthaI) [reply aws/table_aws_inspector2_coverage_statistics.go:36]: Added missing optional key quals along with the json quals.

File: docs/tables/aws_inspector2_coverage.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  resource_id,
+  resource_type,
+  scan_type,
+  scan_status_reason,
+  scan_status_code
+from
+  aws_inspector2_coverage;
+```
+
+### List coverage that are inactive
```
[the remark this answers is not in the document]
Comment (ParthaI) [reply docs/tables/aws_inspector2_coverage.md:21]: Updated the query heading.

File: docs/tables/aws_inspector2_coverage.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  resource_type,
+  ec2_ami_id,
+  ec2_platform
+from
+  aws_inspector2_coverage
+where
+  resource_type = 'AWS_EC2_INSTANCE';
+```
+
+### List coverage of package scan
```
[the remark this answers is not in the document]
Comment (ParthaI) [reply docs/tables/aws_inspector2_coverage.md:51]: Updated the query heading.

File: docs/tables/aws_inspector2_coverage_statistics.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+```sql
+select
+  total_counts,
+  counts_by_group
+from
+  aws_inspector2_coverage_statistics;
+```
+
+### Get group count of coverage statistic
```
[the remark this answers is not in the document]
Comment (ParthaI) [reply docs/tables/aws_inspector2_coverage_statistics.md:18]: Updated.

File: docs/tables/aws_inspector2_member.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+select
+  member_account_id,
+  delegated_admin_account_id,
+  relationship_status,
+  updated_at
+from
+  aws_inspector2_member;
+```
+
+### List members that status has been updated in last 30 days
```
[the remark this answers is not in the document]
Comment (ParthaI) [reply docs/tables/aws_inspector2_member.md:21]: Updated.

**Comment by misraved** [comment]
Addresses https://github.com/turbot/steampipe-plugin-aws/issues/1643

File: aws/table_aws_inspector2_member.go [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsInspector2Member(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_inspector2_member",
+		Description: "AWS Inspector2 Member",
+		List: &plugin.ListConfig{
```
Comment (misraved) [inline aws/table_aws_inspector2_member.go:22]: @ParthaI I think we should add support for only_associated column as an optional qual per https://docs.aws.amazon.com/inspector/v2/APIReference/API_ListMembers.html

**Pull request merged**